NEWS.md

```diff
@@ -36,6 +36,14 @@ Run-time formula syntax
 * Methods for `Base./(::AbstractTerm, ::AbstractTerm)` are added, allowing
   nesting syntax to be used with `Term`s at run-time as well [#470]
 
+MixedModels v3.6.0 Release Notes
+========================
+* Add `likelihoodratiotest` method for comparing non-mixed (generalized) linear models to (generalized) linear mixed models [#508].
+
+MixedModels v3.5.2 Release Notes
+========================
+* Explicitly deprecate non-functional `named` kwarg in `ranef` in favor of `raneftables` [#507].
```

Comment:
```suggestion
* Explicitly deprecate vestigial `named` kwarg in `ranef` in favor of `raneftables` [#507].
```
I was reading `non-functional` as "not a function type of argument" instead of "no longer used argument"

Reply:
Ah, yeah, I agree!